lib/membrane_element_rtp_aac/payloader.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Membrane.RTP.AAC.Payloader do
+  @moduledoc "
+    Payloader wrapping non-encapsulated AAC frames into RTP payloads in accordance with RFC3640.
+  "
+
+  use Membrane.Filter
+
+  alias Membrane.Buffer
+  alias Membrane.{AAC, RTP}
+  alias Membrane.RTP.AAC.Utils
+
+  def_input_pad :input, accepted_format: %AAC{encapsulation: :none}
+  def_output_pad :output, accepted_format: %RTP{}
+
+  def_options mode: [
+                spec: Utils.mode()
+              ],
+              frames_per_packet: [
+                spec: pos_integer()
+              ]
```

Comment:
let's add description of these options (this applies also to the `depayloader.ex`)

Reply:
Should the currently unused options (channels, sample_rate, profile) also provide descriptions?